docs/src/main/tut/freemonad.md

```diff
@@ -238,7 +240,7 @@ recursive structure by:
 This operation is called `Free.foldMap`:
 
 ```scala
-final def foldMap[M[_]](f: S ~> M)(M: Monad[M]): M[A] = ...
+final def foldMap[M[_]](f: NaturalTransformation[S,M])(M: Monad[M]): M[A] = ...
```

Comment:
The build is failing because `~>` is imported, but `NaturalTransformation` is not. We should probably pick either one or the other for the tutorial and stick to it, but I do like your note about one being an alias for the other.